media/css/firefox/new/scene2.less

```diff
@@ -39,8 +39,8 @@
 
 .js .help {
     display: block;
-    padding: 0 190px;
-    margin-bottom: @baseLine * 2;
+    max-width: 40em;
```

Comment:
This is fixing a responsive layout bug I noticed at small screen sizes on scene2 of the old `/firefox/new` template.